.pre-commit-config.yaml

```diff
@@ -11,3 +11,7 @@ repos:
   rev: 6.0.0
   hooks:
   - id: flake8
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: v1.17.0
+  hooks:
+  - id: mypy
```

Comment:
This produces the following error, which I don't know how to solve:

```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 2

djhtml/options.pyi: error: Duplicate module named "djhtml.options" (also at "djhtml/options.py")
djhtml/options.pyi: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#mapping-file-paths-to-modules for more info
djhtml/options.pyi: note: Common resolutions include: a) using `--exclude` to avoid checking one of them, b) adding `__init__.py` somewhere, c) using `--explicit-package-bases` or adjusting MYPYPATH
Found 1 error in 1 file (errors prevented further checking)
```

The error message suggests "adding an `__init__.py` somewhere", but where?